### ISSUE TITLE
pytube - Certificate Verify failed

### ISSUE THREAD

**Issue by deepakchandra00** [pytube/pytube#373]
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:777)>

how can i fix this here is my code.

`import os

from flask_wtf.csrf import CSRFProtect

# Flask Imports
from flask import Flask, request
from flask import render_template
from flask_table import Table, Col
from flask import send_file
from flask import redirect

#Pytube Imports
import pytube

#Helpers
import stream_search as streamSearch

#Launch Flask App
app = Flask(__name__)

SECRET_KEY = os.urandom(32)
app.config['SECRET_KEY'] = SECRET_KEY

csrf = CSRFProtect(app)

@app.route('/', methods=['GET', 'POST'])
def root():
    form = streamSearch.SearchForm(request.form)

    if form.validate_on_submit():
        url = form.url.data
        unique_id_index = url[url.index('?v=')+3::]
        print(unique_id_index)
        return redirect('/get-hd-video'+'?url='+unique_id_index, code=302)
    else:
        print ("Error validating form: ", form.errors)
    return render_template('base.html', form=form)

# route example: downlink.com/get-hd-video?url=
@app.route('/get-hd-video', methods=['GET', 'POST'])
def get_video_streams():
    url = 'http://www.youtube.com/watch?v='+request.args.get('url')
    print(url)
    youtube = pytube.YouTube(url)
    print("Getting youtube streams")
    files = youtube.streams.filter(progressive=True).all()
    stream_table = streamSearch.StreamTable(files)
    print(stream_table)

    return render_template('stream_result', stream_table=stream_table)

@app.route('/download')
def download():
    return redirect("http://www.example.com", code=302)

if __name__ == '__main__':
    app.run(debug=True),  # Do not leave on on real situations`

**Comment by jakedent** [comment]
Try forcing python 3.5 - I believe there are some issues running PyTube with Python 3.6.

**Comment by jakedent** [comment]
There is another solution that has worked for me:

Run an install command shipped with Python 3.6:

```
cd /Applications/Python\ 3.6/
./Install\ Certificates.command
```

OR Install the certifi package with:

`pip install certifi`

If you receive a 403 forbidden error after completing the instructions in the above link, just create a Youtube account and get an API key. 

Hope this helps.
